src/components/User/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { createContext, useContext, useState } from 'react'
+import firebase from 'firebase'
+import { authentication } from '../../firebase'
+
+type UserContext = {
```

Comment:
If you have name collisions here, just leave this as `UserContextType`, because that's what it is. :wink:
```suggestion
type UserContextType = {
```

Reply:
Fixed 👍